guides/v2.2/extension-dev-guide/plugins.md

```diff
@@ -99,7 +99,7 @@ Use the following method names for the `_construct` method in the plugin class:
 
 Magento runs all before methods ahead of the call to an observed method. These methods must have the same name as the observed method with 'before' as the prefix.
 
-You can use before methods to change the arguments of an observed method by returning a modified argument. If there is more than one argument, the method should return an array of those arguments. If the method does not change the argument for the observed method, it should return `null`.
+You can use before methods to change the arguments of an observed method by returning a modified argument. If there is more than one argument, the method should return an array of those arguments. If the method does not change the argument for the observed method, it should return a `null` value.
```

Comment:
To be consistent with the change in next comment (`_after_ methods`), change to: 
`before methods` to `_before_ methods`

Reply:
@dshevtsov, it looks like you're suggesting that we italicize the method names instead of using inline code format. Is that right? That's not consistent with the changes on line 291.

Reply:
I changed all before, after, and around formatting to inline code for consistency.

Reply:
Consistency for inline code format is preferred provided the reference is to the code. In some cases in this topic, it may not be referring to the code, but rather the definition of the method. You'd have to look at the topic holistically to make that kind of decision.

Reply:
Well, this has been, thanks everyone. I'll revert my changes.